terraform/helm/forge/templates/continuous_test.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.continuous_test.enabled }}
```

Comment:
Do we use continuous forge testing today or plan to use in future? I was under impression that this is triggered only on-demand per PR?

Reply:
This is mostly for testing purposes :) We will likely continue to trigger it on-demand manually (e.g. developer CLI command) or by CI. Perhaps we can introduce a nightly job that runs Forge against the tip of `main`, but that will be a CI job as well